Airline_Web_API/Airlina_DB/dbo/Tables/Tickets.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[Tickets]
+(
+	[TicketId] INT IDENTITY(1,1) NOT NULL PRIMARY KEY,
+	[FlightId] INT NOT NULL,
+	[TicketTypeId] INT NOT NULL,
+	[Price] MONEY NOT NULL,
+	[TotalNumber] INT NOT NUll,
+	[TicketsLeft] INT NOT NULL,
```

Comment:
Please review naming of these columns. They are both represent number of tickets but named differently.